game.py

```diff
@@ -12,7 +12,14 @@ class Random(Bot):
     """
     Cory's bot :)
     """
-    pass
+
+    def move(self, state):
+        valid_pile_indexes = [i for i, pile in enumerate(state) if pile >= 1]
+        if not valid_pile_indexes:
+            raise NimException(f'No valid moves on state: {state}')
```

Comment:
Technically, the above 3 lines should be copied and pasted into the `move` method of every Bot we will create.
Any ideas on how we could simplify the process?